tests/console/clamav.pm

```diff
@@ -74,14 +74,12 @@ sub run {
 
     # Clamd start timeout sometimes. The default systemd timeout is 90s,
     # override it with a longer duration in runtime.
-    my $runtime_dir = '/run/systemd/system/clamd.service.d';
-    assert_script_run "mkdir -p $runtime_dir";
-    assert_script_run "echo -e \'[Service]\\nTimeoutSec=400\' > $runtime_dir/override.conf";
+    assert_script_run("sed -i '/;User/i TimeoutSec=30min' /usr/lib/systemd/system/clamd.service");
```

Comment:
how did you conclude to this 30min timeout?

Reply:
Tried incrementing by 10min until I reached a sufficient time.

Reply:
I think that 30min is too big timeout for starting a service. It seems a bit buggy to me